cmd/kubeadm/app/cmd/selfhosting.go

```diff
@@ -127,4 +123,4 @@ func getSelfhostingSubCommand() *cobra.Command {
 	options.AddKubeConfigFlag(cmd.Flags(), &kubeConfigFile)
 
 	return cmd
-}
+}
```

Comment:
files should end with an empty new line.

Reply:
This was corrected once verify and update where run! the PR is not longer in WIP. Thanks for the comments!